Automattic/wp-calypso
# Launchpad: Fix font weight of site url top level domain

#### Proposed Changes

* Updates the top level domain font weight of site url in launchpad

<img width="489" alt="Screen Shot 2022-08-22 at 12 10 49 PM" src="https://user-images.githubusercontent.com/5414230/186000077-f7de0756-53c1-4cd5-a684-b0e57a86594e.png">

#### Testing Instructions



* Check out PR
* `yarn start`
* Navigate to http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=[SITE_SLUG_HERE]
* The top level domain of the site url displayed above the checklist should be bolded like in the figma designs 9wpoYJrMlYKAKUYvVBMN74-fi 

#### Pre-merge Checklist






Related to #
